src/temp/ImagesSlider/ImagesSlider.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement, useEffect } from 'react';
+import Stories from 'react-insta-stories';
+import getImageUrl from '../../utils/getImageURL';
+import styles from './Slider.module.scss';
+import { SliderImage } from '../../features/projects/components/PlantLocation/ImageSlider';
+
+interface Props {
+  images: SliderImage[];
+  height: number | string;
+  imageSize: string;
+  type: string;
+}
+
+const ImagesSlider = ({ images, height, imageSize, type }: Props) => {
+  const [slider, setSlider] = React.useState<ReactElement>();
```

Comment:
The component name ImagesSlider could be more descriptive. Consider naming it based on its functionality or purpose, such as ProjectImageSlider.

Reply:
I'd suggest keeping the name generic, as this images slider can be used in other places. In the project details screen itself, the slider can be used for project images (type="project") and plant location images (type="coordinate"). No reason to restrict the usage.